plugins/inputs/interrupts/README.md

```diff
@@ -13,65 +14,99 @@ The interrupts plugin gathers metrics about IRQs from `/proc/interrupts` and `/p
   ## deployments.
   # cpu_as_tag = false
 
+  ## spurious interrupt counters can be collected
+  # spurious = false
```

Comment:
Let's just enable this by default, and rely on the [metric filtering](https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#metric-filtering) for those who don't want spurious counts per IRQ.

Reply:
ok